compose.yaml

```diff
@@ -34,3 +34,13 @@ services:
       interval: 10s
     env_file: ".env"
     entrypoint: npm run watch
+  test_script:
```

Comment:
Yeah, this won't be necessary. You can run it directly locally. You can also install `jq`. If you're on Windows, you just run (in WSL)

```
$ sudo apt install jq
```